geode-core/src/main/java/org/apache/geode/cache/ExpirationAction.java

```diff
@@ -150,6 +150,21 @@ public static ExpirationAction fromString(String s) {
     return null;
   }
 
+  public static ExpirationAction fromXmlString(String xmlValue) {
```

Comment:
ExpirationAction is an external API so you probably do not want to add this method here. Instead add it on one of the internal classes. External apis need to be documented and can not change in future releases (without first being deprecated and then a major release done). It seems like this method could instead be on one of the internal xml parsing classes. 
If you do end up adding it here then add a javadoc comments that defines the allowed "xmlValue" and that it will throw IllegalArgumentException, etc.

Reply:
comments added